nixos/modules/services/web-apps/crabfit.nix

```diff
@@ -0,0 +1,171 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    literalExpression
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    ;
+
+  inherit (lib.types)
+    attrsOf
+    package
+    port
+    str
+    ;
+
+  cfg = config.services.crabfit;
+in
+
+{
+  options.services.crabfit = {
+    enable = mkEnableOption "Crab Fit, a meeting scheduler based on peoples' availability";
+
+    frontend = {
```

Comment:
I would only put the package option as frontend.package, or maybe even just frontendPackage instead of nesting everything in the frontend.* scope.

Reply:
`finalDrv` is a read-only option

Reply:
Just ignore this comment, I miss read the indentation on the gh web interface thing earlier. It's fine as is.